installed-versions.json

```diff
@@ -1,14 +1,20 @@
 {
+  "version": "6.0.0",
   "name": "idp-profile-ui",
   "dependencies": {
     "@babel/polyfill": {
       "version": "7.12.1",
       "resolved": "https://registry.npmjs.org/@babel/polyfill/-/polyfill-7.12.1.tgz",
       "overridden": false
     },
+    "@sentry/vue": {
+      "version": "7.119.0",
+      "resolved": "https://registry.npmjs.org/@sentry/vue/-/vue-7.119.0.tgz",
+      "overridden": false
+    },
     "@simplewebauthn/browser": {
-      "version": "4.1.0",
-      "resolved": "https://registry.npmjs.org/@simplewebauthn/browser/-/browser-4.1.0.tgz",
+      "version": "9.0.1",
```

Comment:
big jump! How long has it been since the last update?

Reply:
Another time I forgot to run `make depsupdate`. See https://github.com/silinternational/idp-profile-ui/pull/169 for details, but the short story is the repo has ui and server, and most of the changes happen in the server code, so not many breaking changes in the ui code.